packages/smooth_app/lib/helpers/analytics_helper.dart

```diff
@@ -73,15 +71,15 @@ class AnalyticsHelper {
     }
 
     try {
+      MatomoTracker.instance.visitor = Visitor(
```

Comment:
Just asking: shouldn't it be _after_ the call to `initialize`?

Reply:
Oh yes right

Reply:
Updated + a change to the uuid